src/index.js

```diff
@@ -1,42 +1,45 @@
 import './style.css';
+import './modules/checkbox';
 
 const list = document.querySelector('.todo-list');
 
-const myList = [
-  {
-    description: 'wash car',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'homework',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'gym',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'study rails',
-    completed: false,
-    index: 4,
-  },
-];
+let myList = [];
+
+if (localStorage.myList !== undefined) {
+  myList = JSON.parse(localStorage.myList);
+}
 
 function todoTasks() {
   myList.forEach((todo) => {
```

Comment:
- [ ] Kindly ensure `arrow functions ` are used instead of the `function keyword`. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look modern.